system/modules/core/library/Contao/Widget.php

```diff
@@ -1486,6 +1486,12 @@ public static function getAttributesFromDca($arrData, $strName, $varValue=null,
 			}
 		}
 
+		// Warn if someone uses the "encrypt" flag (see #8589)
+		if (isset($arrAttributes['encrypt']))
+		{
+			@trigger_error('Using the "encrypt" flag' . (!empty($strTable) && !empty($strField) ? ' on ' . $strTable . '.' . $strField : '') . ' has been deprecated and will no longer work in Contao 5.0. Use the load and save callbacks to implement your own encryption routine instead.', E_USER_DEPRECATED);
```

Comment:
Same here, please adjust to something like `Use well tested third party libraries such as phpseclib`.